x-pack/plugins/kubernetes_security/public/components/container_name_widget/row_container_name_helper.tsx

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Probably better to name this component `container_name_row.tsx`. We should also have unit tests for this component